backend/router.ts

```diff
@@ -0,0 +1,82 @@
+import { Router, Request, Response} from "express";
+import {DatabaseFunctions} from "./repository";
+import {createHmac} from "crypto";
+
+export class MainRouter {

```

Comment:
It might be worth match class name with file name (in this case file name might be named main-router.ts)